tests/mockServer/mock.js

```diff
@@ -0,0 +1,341 @@
+import express from 'express'
+import bodyParser from 'body-parser'
+import _ from 'lodash'
+
+import frontendSpec from './data/frontendSpec.json'
+import projects from './data/projects.json'
+import projectsSummary from './data/summary.json'
+import artifacts from './data/artifacts.json'
+import featureSets from './data/featureSets.json'
+import features from './data/features.json'
+import featureVectors from './data/featureVectors.json'
+import runs from './data/runs.json'
+import pipelines from './data/pipelines.json'
+import schedules from './data/schedules.json'
+import artifactTags from './data/artifactsTags.json'
+
+// Here we are configuring express to use body-parser as middle-ware.
+const app = express()
+app.use(bodyParser.urlencoded({ extended: false }))
+app.use(bodyParser.json())
+
+// MLRun object Templates
+const projectTemplate = {
+  kind: 'project',
+  metadata: { name: '', created: '', labels: null, annotations: null },
+  spec: {
+    description: '',
+    goals: null,
+    params: null,
+    functions: null,
+    workflows: null,
+    artifacts: null,
+    artifact_path: null,
+    conda: null,
+    source: null,
+    subpath: null,
+    origin_url: null,
+    desired_state: 'online'
+  },
+  status: { state: 'online' }
+}
+const summuryTemplate = {
+  name: '',
+  functions_count: 0,
+  feature_sets_count: 0,
+  models_count: 0,
+  runs_failed_recent_count: 0,
+  runs_running_count: 0
+}
+const projectExistsConflict = {
+  detail: "MLRunConflictError('Conflict - Project already exists')"
+}
+
+// Mock consts
+const mockHome = process.cwd() + '/tests/mockServer'
+const mlrunAPIIngress =
+  '/mlrun-api-ingress.default-tenant.app.ui-dev.iguazio-cd0.com'
+const port = 30000
+
+// Support function
+function makeUID(length) {
+  let result = ''
+  const characters = 'abcdef0123456789'
+  let charactersLength = characters.length
```

Comment:
`charactersLength` does not get reassigned so we can use `const`.
```suggestion
  const charactersLength = characters.length
```